src/Shipment/Concern/EncodesRecipient.php

```diff
@@ -18,6 +22,23 @@ private function encodeRecipient(?ContactDetails $recipient): ?array
             return null;
         }
 
+        $street = $recipient->street;
+        $streetAdditionalInfo = $recipient->streetAdditionalInfo;
+
+        // Remove overlap if extra info starts with the end of street
+        [$street, $streetAdditionalInfo] = $this->removeStreetOverlap($street, $streetAdditionalInfo);
```

Comment:
Waar komt die overlap vandaan, is dat iets dat in de praktijk voorkomt? En is het dan altijd eind vd street die overlapt met begin additional info? (of kan er ook andere data verdubbeld zijn, bedoel ik) (ik zie weleens huisnummers in verschillende velden tegelijk voorbij komen)